template/en/default/account/prefs/account.html.tmpl

```diff
@@ -107,7 +107,7 @@
           <th align="right">Current password:</th>
           <td>
             <input type="hidden" name="old_login" value="[% user.login FILTER html %]">
-            <input autocomplete="current-password" type="password" name="old_password" id="old_password">
+            <input autocomplete="off" type="password" name="old_password" id="old_password">
```

Comment:
Actually after seeing that we already do autocomplete for current password, let's change this back to `current-password` as looking this up that is correct already.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

Reply:
You are right. Both were syntactically correct but keeping "current-password" value here is a feature and yields no bug.